app/assets/javascripts/admin/line_items/controllers/line_items_controller.js.coffee

```diff
@@ -60,6 +60,7 @@ angular.module("admin.lineItems").controller 'LineItemsCtrl', ($scope, $timeout,
     [formattedStartDate, formattedEndDate] = $scope.formatDates($scope.startDate, $scope.endDate)
 
     RequestMonitor.load LineItems.index(
+      "q[order_distributor_name_or_order_bill_address_phone_or_order_bill_address_firstname_or_order_bill_address_lastname_or_variant_product_supplier_name_or_order_email_or_order_number_or_product_name_cont]": $scope.query,
```

Comment:
It would be easier to read to put these fields into an array and join it with `_or_`.

Reply:
I agree, done with: ac739108a2